test/e2e/scaffold/scaffold.go

```diff
@@ -54,6 +53,7 @@ type Options struct {
 	APISIXAdminAPIKey     string
 	EnableWebhooks        bool
 	APISIXPublishAddress  string
+	disableNodeSelector   bool
```

Comment:
Its name should be related to namespace, right?

Reply:
yes